### ISSUE TITLE
"Failed to transform with babel"

### ISSUE THREAD

**Issue by YogliB** [jakedeichert/svelvet#40]
Environment info:
Svelte v3.18.2
Svelvet v0.2.1
OS: Windows 10

Output:
```bash
PS D:\Projects\Svelte\svelvet> npm run dev

> svelvet-template@1.0.0 dev D:\Projects\Svelte\svelvet
> npm run clean && svelvet


> svelvet-template@1.0.0 clean D:\Projects\Svelte\svelvet
> rimraf dist

Copied asset dist/index.html
Svelte compiled dist/App.js
Svelte compiled dist/components/Footer/Links.js
Svelte compiled dist/components/Footer/index.js
Svelte compiled dist/components/Header.js

Building web_modules with snowpack...

Failed to transform with babel: dist/components/Footer/index.js
{ Error: unknown: ENOENT: no such file or directory, open 'D:\Projects\Svelte\svelvet\dist\web_modules\import-map.json' 
    at Object.openSync (fs.js:443:3)
    at Object.readFileSync (fs.js:343:35)
    at readImportMapFile (D:\Projects\Svelte\svelvet\node_modules\snowpack\assets\babel-plugin.js:7:15)
    at getImportMap (D:\Projects\Svelte\svelvet\node_modules\snowpack\assets\babel-plugin.js:27:24)
    at PluginPass.pre (D:\Projects\Svelte\svelvet\node_modules\snowpack\assets\babel-plugin.js:83:28)
    at transformFile (D:\Projects\Svelte\svelvet\node_modules\@babel\core\lib\transformation\index.js:96:27)
    at transformFile.next (<anonymous>)
    at run (D:\Projects\Svelte\svelvet\node_modules\@babel\core\lib\transformation\index.js:35:12)
    at run.next (<anonymous>)
    at Object.transform (D:\Projects\Svelte\svelvet\node_modules\@babel\core\lib\transform.js:27:41)
  errno: -4058,
  syscall: 'open',
  code: 'ENOENT',
  path:
   'D:\\Projects\\Svelte\\svelvet\\dist\\web_modules\\import-map.json' }
```

The Babel error continues to other files as well.

A wierd thing that is happening is that `node_modules\snowpack\dist-node\index.bin.js`  is being opened after `Building web_modules with snowpack...` and the errors are shown only after I close it.

**Comment by jakedeichert** [comment]
I'm on macOS with node v12 and I'm not getting this error at all... it could be a Windows-specific thing. It looks like snowpack is completely failing to run on Windows. 

Can you copy/paste this script into a js file in the root of your project, run it and paste the outputs here?

`run.js`
```js
const path = require('path');

const snowpackLocation = path.resolve(
    require.resolve('snowpack'),
    '../index.bin.js'
);

console.log(
    `${snowpackLocation} --include 'dist/**/*' --dest dist/web_modules`
);
```

Run it with `node run.js`


After i see the output I might have a better idea of what could be happening.

**Comment by moitias** [comment]
Ran into the same / very similar issue on Windows as well. 

At least for me, an indication of this being indeed a (classic) Windows issue with node was the fact that a window asking for how to open a JS file popped up. This is typically caused by the windows semantics of trying to open a file by the default application when executed (rather than the application specified in the hashbang; eg `#!/usr/bin/env node
`), and the default application for .js files is Windows Scripting Host, which, doesn't really do it's job.. 

The fix is just to add `node` before the snowpack binary, eg:
```
        const { stdout, stderr } = await exec(`node ${snowpackLocation} --include 'dist/**/*' --dest dist/web_modules ${maybeOptimize}`);
```

This makes at least the very basic case work for me on Windows, although going further there were problems especially with hot reloading / recompilation (it seems to generate similarly corrupted js files in dist when recompilation is triggered) - will try to investigate more and file an issue.

**Comment by jakedeichert** [comment]
@moitias thank you for the fix!! This makes sense... I had a feeling it had something to do with the way I was invoking snowpack 😅

**Comment by jakedeichert** [comment]
Released as [v0.3.1](https://github.com/jakedeichert/svelvet/releases/tag/v0.3.1)
